spec/controllers/events_controller_spec.rb

```diff
@@ -1,6 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe EventsController, type: :controller do
+  let(:admin_person) { FactoryGirl.build(:person, :admin) }
```

Comment:
Would love to see us add a context for the case where the user is *not* an admin.